doc/source/default-settings.rst

```diff
@@ -413,9 +413,9 @@ Add options
 
         add-folder-name = ``{doc[year]} / {doc[author]}-{doc[title]}``
 
-    If this setting is empty, a (MD5) hash of some document contents followed
-    by the author names is used. This will generally result in unique folder
-    names. If a folder name is not unique and the document does not appear to
+    If this setting is empty, a (MD5) hash of some document contents is used.
```

Comment:
```suggestion
    If this setting is empty, a hash of some document contents is used.
```
? We probably shouldn't be specific about the hash here.

Reply:
I reckon the whole paragraph was a little unclear. I rewrote it with the future human-readable default in mind.